Documentation/rfc/v3api.md

```diff
@@ -191,3 +191,6 @@ WatchResponse {
     …
     
 ```
+
+[api protobuf]: /etcdserver/etcdserverpb/rpc.proto
```

Comment:
I think these two links should probably NOT start with a `/` that fixes them to the root of the HTTP name space. Please check. Relative links should be, e.g., 
`[api-protobuf]: etcdserver/etcdserverpb/rpc.proto`, but in fact, these links need to give absolute URL, since they point not to documents, but to code (a protobuf spec). I think
`[api-protobuf]: https://github.com/coreos/etcd/blob/master/etcdserver/etcdserverpb/rpc.proto` is probably the link you want.